core/src/main/se/tevej/game/view/gui/BuildingGui.java

```diff
@@ -15,13 +15,15 @@
 
 
 public class BuildingGui {
+    private static final int IMAGESIZE = 32;
+    private static final int PADDING = 5;
     private GuiFactory guiFactory;
     private TTable buildingTable;
     @SuppressFBWarnings(
         value = "SS_SHOULD_BE_STATIC",
         justification = "No need to be static and checkbugs will complain if it is."
     )
```

Comment:
Please remove this as the variable it was used on is removed.

Reply:
hm?

Reply:
oh, varningen?

Reply:
Yeah

Reply:
supressen då